### PR TITLE
First pass at creating installation instructions for users

### PR DESCRIPTION
**Pull request by chillypepper**
I had a go at making an installation guide for the website, hopefully will make things a bit easier for new users looking to try OpenGOAL. Some notes about considerations below:

- Included some instructions and screenshots for iso extraction with ImgBurn, as that was the tool linked in the Discord (and I had prior experience :P)
- Exported and re-used the download link from the main page in the docs, felt like an easy approach but I can remove this and link to the main page if that's preferable
- Tried to keep in language reminding people this is a community project and support isn't guaranteed, but open to any changes for this

The instructions for Linux are a bit light as I'm a Windows scrub but I think it covers the important points, and I've included links to the Discord and to Twitch to hopefully get people more involved in the Jak community as well. Happy for any changes to be made to the branch, or to make them myself if you have any requests :D

Cheers, big fan of the project!

### CODE REVIEW

File: documentation/usage/installation.md [hunk 1]
```diff
@@ -2,18 +2,153 @@
 sidebar_position: 0
 ---
 
+import { LauncherDownloadLink } from "/src/pages/index.js"
+
 # Installation
 
+## Opening notes
+
+At the moment OpenGOAL only supports running the first Jak and Daxter game, with Jak 2 and Jak 3 in progress. There are no timelines for their completion, as this is an open source passion project, but you're more than welcome to contribute to their development with us! See [Contributing](/docs/category/contributing) and [Developing](/docs/category/developing) for more information.
```
Comment (xTVaser) [inline documentation/usage/installation.md:11]: let's keep this to just Jak 2 being in progress for now

File: documentation/usage/installation.md [hunk 2]
```diff
@@ -2,18 +2,153 @@
...
+
+### Downloading and Installing OpenGOAL
+
+For Windows users we provide a pre-packaged OpenGOAL installation that you can use to get started right away. The latest download can be found below:
+
+<div className="row markdownMarginBottom">
+    <div className="col col--4">
+        <LauncherDownloadLink />
+    </div>
+</div>
```
Comment (xTVaser) [inline documentation/usage/installation.md:68]: cool that this just works lol

File: documentation/usage/installation.md [hunk 3]
```diff
@@ -2,18 +2,153 @@
...
+  </div>
+</details>
+
+If you choose to launch OpenGOAL once the install is completed, you'll skip right to the next step. If not you'll need to navigate to the location you chose for OpenGOAL to install (by default this location is `C:\Program Files\OpenGOAL-Launcher`), and run `OpenGOAL-Launcher.exe`.
+
+If this opens successfully, you've reached the final stage! Up next we'll be setting up the launcher, including extracting the assets from your `.iso` file.
+
+### Setting up the launcher
+
+This step is very straightforward, but as this project is still in active development you may come across issues along the way. If you hit issues please remember that this is a community led project, and we can't guarantee support. Please first visit the [FAQs](/faq.md) and if you can't find an answer there, you might be able to find help in the OpenGOAL [discord](https://discord.gg/VZbXMHXzWv), but we do not guarantee support.
```
Comment (xTVaser) [inline documentation/usage/installation.md:106]: maybe this would be better put near the top in an admonition, since they may run into problems on early steps as well?

File: documentation/usage/installation.md [hunk 4]
```diff
@@ -2,18 +2,153 @@
...
+
+![Launcher extracting assets and completing compilation](/docs/installation/opengoal_launcher_jd1_compiling.png)
+
+This can take some time, but if you find it taking more than a few minutes try opening the `INSTALLATION LOGS` view below the progress bar, and see if there is still progress being made. If you don't see any changes here after a minute or two more, try closing and re-running the launcher.
+
+This process will complete loading the assets from your `.iso` file, and once it's done you'll be ready to run the game!
+
+![Starting Jak and Daxter in OpenGOAL](/docs/installation/opengoal_launcher_jd1_ready_to_start.png)
+
+Once you've reached this screen, you're all set up and ready to go! Just hit `PLAY` to start the game. The other controls are not something you'll need to play the game, as most of them are tools for developers, but do take note of the `SETTINGS AND SAVES` button if you want a link to your saves to move them or back them up, and of course the `UNINSTALL` button if you choose to do so down the line. The OpenGOAL launcher can also be uninstalled from the OpenGOAL installer used earlier in the instructions.
```
Comment (xTVaser) [inline documentation/usage/installation.md:122]: it'd be nice to include a screenshot of an example of requirements not being met, as this seems to confuse some users even though it's fairly obvious (you dont meet the requirements).  Here's a recent example that could be used:
image

File: documentation/usage/installation.md [hunk 5]
```diff
@@ -2,18 +2,153 @@
...
+**ENJOY!**
 
 ## Linux
 
-TODO
+The instructions for Linux users are a bit more sparse than the Windows users, but most of the steps are identical.
+
+To begin with you'll first need to create a `.iso` file from your copy of Jak and Daxter, reading the dics for its content. The [Windows ISO instructions](installation#creating-your-iso) have some example instructions using a Windows program called ImgBurn, which you'll need an alternative to, but the rest of the instructions are the same.
+
+The biggest change is how you run OpenGOAL, as we don't provide a pre-built Linux installer, so the instructions for building the game are below.
```
Comment (xTVaser) [inline documentation/usage/installation.md:134]: we will have a release of the launcher that works for linux soon :tm: but yeah this is accurate for now

File: documentation/usage/installation.md [hunk 6]
```diff
@@ -2,18 +2,153 @@
...
+
+### Running OpenGOAL on Linux
+
```
Comment (xTVaser) [inline documentation/usage/installation.md:137]: most of the questions for linux support tend to involve the steamdeck, as these are linux users that aren't used to linux.  It might be worth adding a steamdeck section, in which case this is a pretty good video tutorial that i believe is still accurate - https://youtu.be/Cv7mlCSHKxc?t=564

File: documentation/usage/installation.md [hunk 7]
```diff
@@ -2,18 +2,153 @@
...
+As there is no pre-built launcher for Linux you will need to instead fetch the latest release from our [GitHub repo](https://github.com/open-goal/jak-project/releases). Here you can choose either the `.bin` installer or extract the files yourself from the `.tar.gz` folder to get the tools for this setup.
```
Comment (xTVaser) [inline documentation/usage/installation.md:138]: the `.bin` isn't an installer it's for an unrelated code editor tool.  They'll want the `.tar.gz`

File: documentation/usage/installation.md [hunk 8]
```diff
@@ -2,18 +2,153 @@
...
+
+If you have some technical experience, and would prefer to build this application from the source yourself, then take a look at the [project README instructions](https://github.com/open-goal/jak-project#setting-up-a-development-environment) instead. Support for this however is outside the scope of this document.
+
+If you're continuing with the builds from the latest releases, either install the `.bin` or extract the `.tar.gz` content to get an install with a similar list of files to the below.
```
Comment (xTVaser) [inline documentation/usage/installation.md:142]: same here, omit talking about the `.bin`

File: documentation/usage/installation.md [hunk 9]
```diff
@@ -2,18 +2,153 @@
...
+
+Once you have these files extracted, there are two main steps required. The first is to extract all of the assets from the `.iso` file you created earlier, using the `extractor` from the install above (eg `./extractor <path-to-iso>`). Once this extraction completes, you should be able to run `gk` from the same folder to start the game.
+
+That's everything! At this point you should have a working copy of the game to play. Join us in the [discord](https://discord.gg/VZbXMHXzWv) or finding streams/streaming yourself on [Twitch](https://www.twitch.tv/directory/game/Jak%20and%20Daxter:%20The%20Precursor%20Legacy) to get involved in the community.
 
 ## MacOS
 
 :::danger
-We do not support MacOS at this time
+We do not officially support MacOS at this time. You may have some success with the [Linux installation instructions](installation#linux), but we will not provide any support if this does not work.
```
Comment (xTVaser) [inline documentation/usage/installation.md:153]: They will have no success!  macOS doesn't support the version of OpenGL we use, and if they are on an Apple Silicon device, we don't support Rosetta/ARM.  There are several people that want to support macOS in the future but yeah, currently unsupported

**Review by xTVaser — CHANGES_REQUESTED** [review]
Overall this looks really good, thanks for fleshing this page out.  Just some minor adjustments and some suggestions for additions.

File: documentation/usage/installation.md (hunk 3 above)
Comment (chillypepper) [reply documentation/usage/installation.md:106]: There is a copy of the warning with links to the FAQs and Discord up in the second paragraph of the `Opening notes` section, but I've added it into a highlight block and made it the first paragraph to make it more obvious. I've made that change for the commit coming next and left this paragraph as is, but if you'd like me to remove it here or make any other changes with the top paragraph once I've pushed it let me know

Comment (xTVaser) [reply documentation/usage/installation.md:106]: I'll leave that up to you, repeating this kind of info isn't a bad thing imo cause it's often missed.

File: documentation/usage/installation.md (hunk 5 above)
Comment (chillypepper) [reply documentation/usage/installation.md:134]: Should I maybe change it to something like `... we don't currently provide a pre-built Linux installer, but keep an eye out for new releases as one will be provided soon. In the mean time, the instructions ...`? I'll leave this as is for now but happy to add an extra note in there if it's planned soon and users might want to reinstall to get the launcher when it's ready

Comment (xTVaser) [reply documentation/usage/installation.md:134]: I'm fine with leaving it as is and then we can update the page once it's actually out there

File: documentation/usage/installation.md (hunk 9 above)
Comment (chillypepper) [reply documentation/usage/installation.md:153]: Ah that's a shame, I've gotten used to things built for linux often just working on macs, but I suppose most of those aren't really GUI tools :P I've changed this note to

`We do not support macOS at this time. Unfortunately macOS does not currently support the version of OpenGL we use for this project, and we additionally do not support any ARM processors, meaning significant changes would be required.`

for the next commit as I think that's good info to have, but I'm happy to just revert back to the original unsupported message if that's preferrable

File: documentation/usage/installation.md (hunk 6 above)
Comment (chillypepper) [reply documentation/usage/installation.md:137]: Yeah wow that's a great video, for the intro to the project as well as the steamdeck setup. I think the only difference in the setup would be that the other discs should now be supported which I'll include as a text note with the link, but the rest still all seems good to go!

File: documentation/usage/installation.md (hunk 4 above)
Comment (chillypepper) [reply documentation/usage/installation.md:122]: Cheers I've now included this image and warning at the top of the setup instructions, hopefully it'll help catch people early! Thanks for this :D

**Comment by chillypepper** [comment]
Just pushed the update for all of these changes, thanks for the comprehensive review! If there's anything else I can improve let me know :)

**Review by xTVaser — APPROVED** [review]
Looks good to me, thanks again
